### DISCUSSION TITLE
generating tldr pages using a language model

### DISCUSSION THREAD

**Discussion by Galz648** [tldr-pages/tldr#10111] [Ideas]
Hi everyone,

I hope you're all doing well! I had an idea that I wanted to share with the community here. I've been using the tldr  cli utility for a while now. 

So, I was thinking, wouldn't it be cool if we could use a language model to generate TLDR pages? I think this could really expand the project

I don't have a lot of experience with language models myself, but I thought it might be a fun project to work on with others in the community. What do you all think? Would anyone be interested in collaborating on this idea? Or does anyone have any suggestions for how we might approach this?

Looking forward to hearing your thoughts!

**Comment by kbdharun** [comment]
I appreciate the suggestion, we could automate the fetching and creation of tldr pages using the language model's suggestions, but it will significantly increase the work for Maintainers as we need to cross-check every command to check if it is accurate. We had a similar stance to a user some time ago who created a massive PR using Deepl for translating pages not having translations, we didn't accept it and suggested that it be implemented client-side, instead of having non-proofread translation here in the repo.

Unlike translations, this is something we could review so probably it will be fine, but I would love to hear other maintainer's opinions regarding the matter.

**Comment by Galz648** [comment]
A client-side implementation of language model generated content could be driven by user reviews. If the content receives a sufficient number of positive reviews, it could be suggested as a potential addition to the repository.

**Comment by Rokiers** [comment]
Waiting for proofreading and translation is to slow
i think we can use this way to keep the original text in the Translation document
like this

- english text
- translate text
`Command usage`
in this way the user can check translate text by themselves，

**Comment by spageektti** [comment]
But what about copyright and license?

**Reply by kbdharun** [reply]
> But what about copyright and license?

Yeah, that is something of concern too as there aren't any well-defined copyright laws dealing with generated model outputs yet.

As mentioned in another similar discussion, we may not accept unverified AI-assisted pages/translations in our repository due to these issues and concerns; but clients are free to implement them as additional features if needed (but with clear distinction that it is machine/AI generated).
